assignments/this.js

```diff
@@ -1,26 +1,55 @@
 /* The for principles of "this";
-* in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
-* write out a code example of each explanation above
-*/
+ * in your own words. explain the four principle for the "this" keyword below.
+ *
+ * 1. creates new function permanently bound to "this" value
```

Comment:
Global or window biding will bind to the largest scope of the project which encompasses everything.